client/src/components/all-components/carousel.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { images } from "src/constants/school_images.js";
+
+function display_images() {
+  return (
+    <>
+      {images.map((item, id) => (
+        <img
+          className="tw-max-w-40 lg:tw-max-w-sm tw-max-h-sm tw-m-5"
+          key={id}
+          src={require("../../assets/images/" + item.img + ".png")}
+          alt={item.alt}
+        />
+      ))}
+    </>
+  );
+}
+
+function display_images_reverse() {
+  return (
+    <>
+      {images.map((item, id) => (
+        <img
+          className="tw-max-w-40 lg:tw-max-w-sm tw-max-h-sm tw-m-5 tw-self-center"
+          key={id}
+          src={require("../../assets/images/" + item.img + ".png")}
+          alt={item.alt}
+        />
+      ))}
+    </>
+  );
+}
+
+const Carousel = () => {
+  return (
+    <>
+      <div className="tw-m-auto tw-relative tw-items-center tw-h-1/6 lg:tw-h-1/4 tw-w-full lg:tw-w-3/4 tw-overflow-hidden">
+        <div className="tw-w-full tw-h-full tw-scroll-smooth tw-animate-infinite-scroll tw-whitespace-nowrap">
+          {display_images()}
+          {display_images()}
```

Comment:
At first I was confused why this is here twice but when i tested it with one i see that the carousel looks empty